CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ Starting with the 0.16.4 release on March 5, 2024, the format is based on
 We follow https://keepachangelog.com/en/1.1.0/. Please add to relevant subsections
 here on every PR where this is applicable.
 
+### Added
+
+* The wandb-core downloader now writes into and restores from the cache unless `skip_cache` is set to `True` by @moredatarequired in https://github.com/wandb/wandb/pull/7364
```

Comment:
Would this also be an accurate description?

> `wandb-core` supports the `skip_cache` option to `Artifact.add_file()`

I don't think users will know what the "wandb-core downloader" is, since that's just an internal file.